apps/explorer/src/components/checkpoints/CheckpointsTable.tsx

```diff
@@ -39,13 +39,20 @@ export function CheckpointsTable({
         rpc.getLatestCheckpointSequenceNumber()
     );
 
+    const checkpoints = useGetCheckpoints(initialCursor, limit);
+
     const count = useMemo(() => {
-        if (maxCursor && initialCursor)
+        if (maxCursor && initialCursor) {
             return Number(initialCursor) - Number(maxCursor);
+            // Special case for ongoing epoch
+        } else if (checkpoints.data && maxCursor) {
+            return (
+                Number(checkpoints.data.pages[0].data[0].sequenceNumber) -
```

Comment:
is there any chance `pages[0]` or `pages[0].data[0]` could be empty?

Reply:
added an error check!